test/m1_facilitator/SeedDataInitialiser.test.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+
+import BigNumber from 'bignumber.js';
+
+import assert from '../test_utils/assert';
+import Gateway, { GatewayType } from '../../src/m1_facilitator/models/Gateway';
+import Repositories from '../../src/m1_facilitator/repositories/Repositories';
+import SeedDataInitialiser from '../../src/m1_facilitator/SeedDataInitialiser';
+
+
```

Comment:
2 new lines are not needed. Only single new lines is fine.

Reply:
Removed the extra line.